server/src/apub/user.rs

```diff
@@ -43,6 +43,9 @@ impl ToApub for User_ {
 
     Ok(person.extend(actor_props).extend(self.get_public_key_ext()))
   }
+  fn to_tombstone(&self) -> Result<Tombstone, Error> {
+    unimplemented!()
```

Comment:
This should be really straightforward, just copy it from Community.

Reply:
There's no concept of deleting users right now, even locally.